manager/controllers/app/moduleinstance.go

```diff
@@ -16,6 +16,7 @@ import (
 	local "fybrik.io/fybrik/pkg/multicluster/local"
 	"fybrik.io/fybrik/pkg/serde"
 	"fybrik.io/fybrik/pkg/storage"
+	openapiclientmodels "fybrik.io/fybrik/pkg/taxonomy/model/base"
```

Comment:
I think that we need a better name than `openapiclientmodels`. Suggestions? :)

Reply:
can we do this refactoring in another PR as it requires changes in many other places ? We can make a PR for this change and also any other changes to the taxonomy. Thanks.

Reply:
Sure